We want /bin/sh left as the shell, not bash for example
Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>
Upstream-Status: Inappropriate [fixed differently in new versions]

Index: gzip-1.3.12/Makefile.am
===================================================================
--- gzip-1.3.12.orig/Makefile.am
+++ gzip-1.3.12/Makefile.am
@@ -55,7 +55,6 @@ gzip.doc.gz: gzip.doc
 SUFFIXES = .in
 .in:
 	sed \
-		-e 's|/bin/sh|$(SHELL)|g' \
 		-e 's|BINDIR|$${GZIP_BINDIR-'\''$(bindir)'\''}|g' \
 		-e 's|[@]VERSION@|$(VERSION)|g' \
 		$(srcdir)/$@.in >$@
